cculkin8/cc-capstone
# cc-holding

# Description
MVP has been hit, all buttons work as they should and the dropdowns populate the correct information now!
Fixes # (issue)
Drop downs are populating and opening properly, edit buttons take you to the right place!

# Testing Instructions
Please create a profile, login, create a post. Like it, then check the homepage. It should appear under the "top posts" section. Also check the random button. Go around and just mess with everything else, login, edit profiles, edit posts. Go to town
# Checklist:
